pdns/dnsdist.cc

```diff
@@ -1165,6 +1165,10 @@ static bool isUDPQueryAcceptable(ClientState& cs, LocalHolders& holders, const s
     dest.sin4.sin_family = 0;
   }
 
```

Comment:
I don't follow this part. In the code above you say you want to set port to 0 in some cases, but here it gets re-assigned to a nonzero value in those cases.

Reply:
I think you are right: we want to set a fallback for `origDest`, because it might confuse the users, but `hopLocal` should never be set to that fallback, and I changed that by moving that block earlier. I'll fix it and a comment explaining what is going on. Thanks!